core-processor/src/ext.rs

```diff
@@ -1211,6 +1264,184 @@ mod tests {
         assert_eq!(0, allowance);
     }
 
+    #[test]
+    /// This function tests:
+    ///
+    /// - `send_commit` on valid handle
+    /// - `send_commit` on invalid handle
+    /// - `send_commit` on used handle
+    /// - `send_init` after limit is exceeded
+    ///
```

Comment:
No need for ending `///`

```suggestion
    /// This function tests:
    ///
    /// - `send_commit` on valid handle
    /// - `send_commit` on invalid handle
    /// - `send_commit` on used handle
    /// - `send_init` after limit is exceeded
```

Reply:
Same for all the tests below.